### PR TITLE
Some suggested changes

### PR DESCRIPTION
**Pull request by mappin**
User may have different mongoDB password.  Suggest adding this line.

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -47,4 +47,6 @@ def read_config(filename, verbose=True):
         password = m.get('password')
         if password.startswith('$'):
             password = os.getenv(password[1:])
+        else:
+            password = m.get('password')
```
Comment (ozacas) [inline src/utils.py:51]: Not handled by line 47?

File: README.md [hunk 2]
```diff
@@ -81,7 +81,8 @@ python3 manage.py runserver # run on local dev. host
 
   # run periodically (eg. quarterly) and at setup to update list of securities/companies/fundamentals
   python3 src/asxtrade.py --want-companies
-  python3 src/asxtrade.py --want-securities
+  python3 src/asxtrade.py --want-isin
+  python3 src/asxtrade.py --want-prices
```
Comment (ozacas) [inline README.md]: Renamed --want-securities to --want-isin and moved --want-prices further down as you typically want to fetch prices each day.

File: src/asxtrade.py [hunk 3]
```diff
@@ -82,7 +82,7 @@ def update_isin(db, config, ensure_indexes=True):
         )
 
     df = None
-    with tempfile.NamedTemporaryFile() as content:
+    with tempfile.NamedTemporaryFile(delete=False) as content:
```
Comment (ozacas) [inline src/asxtrade.py:85]: Ugh... dont want this on non-Windows systems. I'll use platform and only specify it for Windows.

File: src/config.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
    "asx_companies": "https://www.asx.com.au/asx/research/ASXListedCompanies.csv",
-   "asx_isin": "https://www.asx.com.au/programs/ISIN.xls",
+   "asx_isin": "https://www2.asx.com.au/content/dam/asx/issuers/ISIN.xls",
```
Comment (ozacas) [inline src/config.json:3]: Forces a small change to asxtrade.py to handle the data format change, but looks like an improved link - thanks! Updated.

File: src/asxtrade.py (hunk 3 above)
Comment (ozacas) [reply src/asxtrade.py:85]: I've updated asxtrade.py to use platform.system() == 'Windows' and add to kwargs as required

File: src/utils.py (hunk 1 above)
Comment (mappin) [reply src/utils.py:51]: Yes

File: src/asxtrade.py (hunk 3 above)
Comment (mappin) [reply src/asxtrade.py:85]: All good.

**Pull request closed without merging**